src/rulesets/oas3/index.json

```diff
@@ -83,7 +83,20 @@
       "message": "{{error}}",
       "recommended": true,
       "type": "validation",
-      "given": "$..[?(@.example)]",
+      "given": "$..[?(@.example && !@.type)]",
+      "then": {
+        "function": "schemaPath",
+        "functionOptions": {
+          "field": "example",
+          "schemaPath": "$.schema"
+        }
+      }
+    },
+    "valid-type-example": {
```

Comment:
Should it be a recommended rule, or no?
I'd rather disabled by default.

Reply:
We can make it not recommended until we are more confident with it, and bump it up in v5?